README.md

```diff
@@ -3,46 +3,46 @@ Mooltipass Project Repository
+We, therefore, offer the following advantages:  
 - **Stronger security**: we reduce the number of attack vectors by basically having our device type your passwords for you. 
 - A **non-proprietary device**: as our product is open anyone can develop new tools for it. There will never be fees for the services we offer.  
 - An **open-source platform**: all our source code can be viewed, allowing you to check that your credentials are only kept inside the device and not leaked to the outside  
 - A **trusted platform**: as only our tested source code is running on the Mooltipass, there can't be any viruses or malicious programs compromising your stored credentials  
 
 The Platform
 ------------
-The Mooltipass is composed of the main device shown above and a smartcard.  
-On the device are stored your AES-256 encrypted credentials. The smartcard is a read protected memory that needs a PIN code to unlock its contents (AES-256 key + a few websites credentials). As with your chip and pin card, too many tries will permanently lock the smart card.
+The Mooltipass is composed of the main device shown above and a smart card.  
+On the device are stored your AES-256 encrypted credentials. The smart card is a read protected memory that needs a PIN code to unlock its contents (AES-256 key + a few websites credentials). As with your chip and pin card, too many tries will permanently lock the smart card.
 
 The Firmware
 ------------
 **We want the device to be as simple as possible.** Ideally, the end user shouldn't have to spend more than a few seconds to use its basic functionalities.  
-A browser extension runs on the user's computer and sends the current website to the Mooltipass. When the user has to login, the Mooltipass will light up and ask for confirmation to enter the credentials.
+A browser extension runs on the user's computer and sends the current website to the Mooltipass. When the user has to log in, the Mooltipass will light up and ask for confirmation to enter the credentials.
 
 Data Safety
 -----------
-Safety is a primary concern for the Mooltipass development team, which is why we offer several **secure** ways to backup your credentials.  
+Safety is a primary concern for the Mooltipass development team, which is why we offer several **secure** ways to back up your credentials.  
```

Comment:
backup is the correct spelling

Reply:
This is not about spelling. "Backup" is a noun, "to back up" is a verb. You are using a noun where you should be using a verb. See the 2nd entry here: https://www.merriam-webster.com/dictionary/backup

The original poster is right, there is no such thing as "to backup".